Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:21.04 AS build
+
+RUN dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y curl
+
+RUN curl -L -o /usr/local/bin/red https://static.red-lang.org/dl/auto/linux/red-latest && \
+  chmod +x /usr/local/bin/red
+
+# WORKDIR /opt/rebol
+
+# RUN curl -L -O http://www.rebol.com/downloads/v278/rebol-core-278-4-10.tar.gz
+# RUN tar -xzf rebol-core-278-4-10.tar.gz
+# RUN chmod +x rebol-core/rebol
+# RUN cp rebol-core/rebol /usr/local/bin/rebol
+
+# WORKDIR /opt/red
+
+# RUN curl -L -O https://github.com/red/red/archive/refs/tags/v0.6.4.tar.gz
+# RUN tar -xzf v0.6.4.tar.gz
+# RUN cd red-0.6.4
+# RUN echo 'Rebol[] do/args %red.r "-d -r --no-view %environment/console/CLI/console.red"' | /usr/local/bin/rebol +q -s
+
+FROM ubuntu:21.04 AS runtime
+
+RUN dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y \
+  libc6:i386 \
+  libcurl4:i386 \
+  libgdk-pixbuf2.0-0:i386
+
+COPY --from=build /usr/local/bin/red /usr/local/bin/red
+
+WORKDIR /opt/test-runner
+
+# # Pre-compile binaries
+RUN /usr/local/bin/red --no-console --no-view build libRed
```

Comment:
This pre-compiles the runtime which bakes this into the Docker image.

Once the test runner itself is working, we can even pre-compile that into a binary instead of having `Red` interpret it as a script.

Reply:
I don't quite get, where did `build` and `libRed` come from?

Reply:
I used that to force the compiling of the Red library and the runtime, as otherwise they would be just-in-time compiled which would make running the test runner very slow.

Reply:
Oh! I just didn't know about the `build libRed` command. Seems you're more into Red than me now :-D I thought `build` might be a script without file extension or a variable carrying some commandline args created by Docker's dark magic.